### DISCUSSION TITLE
Reading in any PDF, differentiating tables and text

### DISCUSSION THREAD

**Discussion by zejian99** [jsvine/pdfplumber#1009] [Q&A]
Hey developers of this library, I just got started trying out this library and there are lots of promising features. I am curious how do I read in an entire PDF, including text as well as tables all at once? 

So i realised that i only used `extract_text()` function, but there are tables in the PDF with text inside, how do I account for these as well?

There are no specific pages that I can reference since I want to parse many PDFs into this library

**Comment by cmdlineluser** [comment]
Hi @zejian99 

There was a recent question which sounds relevant and may help: https://github.com/jsvine/pdfplumber/discussions/1005
